index.js

```diff
@@ -1,21 +1,15 @@
 var loaderUtils = require('loader-utils');
 var MessageFormat = require('messageformat');
 
-module.exports = function(content) {
-  var query = loaderUtils.parseQuery(this.query);
-  var locale = query.locale || 'en';
-  var messages = (Array.isArray(this.inputValue) && typeof this.inputValue[0] === 'object') ? this.inputValue[0] : this.exec(content, this.resource);
```

Comment:
Actually, now that I think about it, I think their intent is for a resource to remain in the original format (in this case, a pure JSON string) throughout all the transforms, and then only at the end be changed into JS. We could leave this as you have it (messageformat-loader expects a JSON string), and I'll switch `multi-json-loader` to output a JSON string instead of JS. Does that make sense?